annual-chapter-report.form

```diff
@@ -0,0 +1,20 @@
+# PyLadies Annual Chapter Activity Report
+
+Please share with us summary of your group's activities in the past year, and celebrate!
```

Comment:
*nit*:

`Please share with us a summary of your group's activities from the past calendar year. We're collecting this information so that we can help highlight and celebrate our community.`